compiler/crates/relay-typegen/tests/generate_typescript/fixtures/updatable-operation-plural-field-no-spreads.expected

```diff
@@ -0,0 +1,18 @@
+==================================== INPUT ====================================
+query UpdatableQuery @updatable {
+  nodes(ids: [4]) {
+    id
+  }
+}
+==================================== OUTPUT ===================================
+export type UpdatableQuery$variables = {};
+export type UpdatableQuery$data = {
+  get nodes(): ReadonlyArray<{
+    readonly id: string;
+  } | null> | null;
+  set nodes(value: []);
```

Comment:
Flow has an explicit `void` return value here. Should we have that (`undefined`) here as well?

Reply:
I don't think it would make a difference, but I could be wrong.